logos/logos-orchestrator/src/logos/main.py

```
@@ -3204,27 +3257,30 @@ async def auth_parse_log(request: Request, use_profile_auth: bool = False):
+    client_ip = get_client_ip(request)
+    auth = authenticate_api_key(headers) if use_profile_auth else None
+
+    # OpenAI-compatible audio uploads use multipart/form-data. Other inference
+    # operations retain the existing JSON request contract.
+    if is_audio_upload_path(request.url.path):
+        body = await parse_audio_upload(request)
+    else:
+        try:
+            body = await request.json()
+        except json.JSONDecodeError:
+            raise HTTPException(status_code=400, detail="Invalid JSON body")
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)